filip/semantics/vocabulary/vocabulary.py

```diff
@@ -0,0 +1,493 @@
+import operator
+from enum import Enum
+
+from pydantic import BaseModel
+from . import *
+from typing import List, Dict, Union, Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from filip.semantics.vocabulary_configurator import LabelSummary
+
+
+class IdType(str, Enum):
+    class_ = 'Class'
+    object_property = 'Object Property'
+    data_property = 'Data Property'
+    datatype = 'Datatype'
+    relation = 'Relation'
+    combined_relation = 'Combined Relation'
+    individual = 'Individual'
+    source = 'Source'
+
+
+class LoggingLevel(str, Enum):
+    """LoggingLevel for parsing statements"""
```

Comment:
why not use standard loglevels hereß

Reply:
These are not for output on the console, but create separate entries that are saved and can later be viewed by the user when inspecting the vocabulary. Therefore the separation from the classical LoggingLevel

Reply:
please double check this LogLevel Enum, maybe we should integrate it somewhere 

tests/config.py